new/LIT6977PPQusl.xml

```diff
@@ -0,0 +1,71 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6977PPQusl" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer for the healing of wounds</title>
+                <title xml:lang="gez" xml:id="t2">ጸሎተ፡ ቁስል፡ አመ፡ ረገዝዎ፡ ለእግዚእነ፡ የማናይ፡ ገቦሁ፡</title>
+                <title xml:lang="gez" corresp="#t2" type="normalized">Ṣalota qusl ʾama ragazǝwwo la-ʾǝgziʾǝna yamānāy gabbohu</title>
```

Comment:
I would prefer ቍስለ፡ in the title, because this word can be found in Dillmanns online lexicon.